src/dotnet/commands/dotnet-new/CSharp_Web/project.json.template

```diff
@@ -0,0 +1,108 @@
+{
+  "userSecretsId": "aspnet-WebApplication-0799fe3e-6eaf-4c5f-b40e-7c6bfd5dfa9a",
+
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-3002702",
```

Comment:
Spoke to @sayedihashimi @mlorbetske. We are going to have to manually update the project.json when we have the next `-final` builds.